cvat/apps/auto_annotation/import_modules.py

```diff
@@ -0,0 +1,36 @@
+import ast
+from collections import namedtuple
+import importlib
+
+Import = namedtuple("Import", ["module", "name", "alias"])
+
+def import_modules(source_code: str):
+    results = {}
+    imports = parse_imports(source_code)
+    for import_ in imports:
+        module = import_.module if import_.module else import_.name
+        loaded_module = importlib.import_module(module)
+
+        if not import_.name == module:
+            loaded_module = getattr(loaded_module, import_.name)
+
+        if import_.alias:
+            results[import_.alias] = loaded_module
+        else:
+            results[import_.name] = loaded_module
+
+    return results
+
+def parse_imports(source_code: str):
+    root = ast.parse(source_code)
+
+    for node in ast.iter_child_nodes(root):
+        if isinstance(node, ast.Import):
+            module = []
+        elif isinstance(node, ast.ImportFrom):
+            module = node.module
+        else:
+            continue
+
+        for n in node.names:
+            yield Import(module, n.name, n.asname)
```

Comment:
To support `import com.stackoverflow` probably it is necessary to modify source code a little bit. Right? Did you get the answer from the stakeoverflow?
`yield Import(module, n.name.split('.'), n.asname)`

https://stackoverflow.com/questions/9008451/python-easy-way-to-read-all-import-statements-from-py-module

Reply:
The code is adopted from the linked stackoverflow question, yes.

`importlib.import_module` expects the `com.stackoverflow` string instead of a list of strings, so there's no benefit for splitting on the `.` operator for this code that I can tell.